hack/jenkins/e2e-runner.sh

```diff
@@ -82,6 +82,12 @@ function fetch_server_version_tars() {
     unset CLUSTER_API_VERSION
 }
 
+function fetch_gci_version_tars() {
+    echo "Using GCI builtin version: ${GCI_K8S_VERSION}"
```

Comment:
Split `setup_gci_vars` into two functions and thus didn't have to use that global variable. PTAL.